composer.json

```diff
@@ -29,7 +29,7 @@
         "ext-json": "*",
         "ext-dom": "*",
         "ext-simplexml": "*",
-        "phpunit/phpunit": "~4.0|~5.0",
+        "phpunit/phpunit": "~4.8",
```

Comment:
Looks like the underscore-namespaced class names were dropped from PHPUnit 6. `^4.8.35|^5.4.0` should avoid the failures that were reported for PHP 7 and 7.1. Alternatively, you could migrate to the new test class naming convention (e.g., replace `PHPUnit_Framework_TestCase` with `\PHPUnit\Framework\TestCase`).